src/actions/guides/frontend/rendering_html.cr

```diff
@@ -140,11 +140,11 @@ class Guides::Frontend::RenderingHtml < GuideAction
     end
     ```
 
-    If you need to pass in data attributes, or any arbitrary attributes for use in SPAs (i.e. ng-app, v-bind:click, etc...), you can also use a string.
+    If you need to pass in data attributes, or any arbitrary attributes for use in SPAs (i.e. ng-app, v-on:click, etc...), you can also use a string.
 
     ```crystal
     def content
-      div(ng_model: "something", data_action: "someAction", "v-bind:click": "update")
+      div(ng_model: "something", data_action: "someAction", "v-on:click": "update")
```

Comment:
Ha! Good catch. It's been a long time since I used angular 😄

Reply:
The `v-on:click` and `v-bind` is actually Vue.js syntax. I've changed to `v-on` since `Vue.js was mentioned in other place with `v-bind:click`. I think in Angular 1 there was `ng-bind` and I think `ng-click`. Updated to Angular attrs here and left `v-on:click` in another comment mentioning Vue.js

Or maybe update this to Vue.js too since Angular 1 is kind of old, so why mention in leading edge framework? 😄

Reply:
I probably confused the two 😅 It doesn't really matter which framework just as long as it shows that you can do custom attributes this way. I think your change here is fine 😄